### ISSUE TITLE
Не хватает возможности играть в шахматы

### ISSUE THREAD

**Issue by FloatingMaster** [lev-veshnyakov/checkers#1]
А в коде есть комментарии на эту тему

**Comment by lev-veshnyakov** [comment]
Right, it was designed to be easily extended to chess, but at the moment it's not ready. I don't know, will I do this, or leave it so as it is yet.
